lib/styles/styles.dart

```diff
@@ -40,7 +41,10 @@ class AppStyle {
   late final _Text text = _Text(scale);
 
   /// Animation Durations
-  final _Times times = _Times();
+  late final _Times times = _Times(disableAnimations);
+
+  // Custom durations / delays
+  late final _CustomTime customTime = _CustomTime(disableAnimations);
```

Comment:
It doesn't look like this is used much, could you use this to fix any remaining issues, and then send it for review again? Mostly I'm trying to evaluate whether it's worth it to keep the `maybeAnimate` + customTime, or just fully revert to custom time. 

A couple of other notes
* Feel free to restore the `delay` if it's really needed to avoid any bugs
* My main concern with the custom time was just the length of the syntax, `150.ms` goes to `styles.customTimes.duration(150)`, can we come up with some alternate syntax that doesn't lose as much readability? Maybe some method extensions on integer...?